scss/events-map.scss

```diff
@@ -512,7 +512,7 @@ article#events {
                 margin-right: 4px;
               }
 
-              .map-red-circle, .circle-button {
+              .map-red-circle, .map-purple-circle, .circle-button {
```

Comment:
seems like the class here should just be `map-circle` or `circle-button`? and then the colors can just be subclasses or whatever?

Reply:
usually would make the effort to refactor that into a base class and sub-class but not worth effort here.